src/mineflayer/Bot.mjs

```diff
@@ -37,8 +38,14 @@ class Bot {
    * safety checks are performed (see below within function implementation for
    * details).
    * @param {String} message
+   * @param {Number} requiredVerbosity necessary verbosity setting to send message, defaults to `VerbosityLevel.Full`
    */
-  chat(message) {
+  chat(message, requiredVerbosity = VerbosityLevel.Full) {
+    if (
+      this.verbosityLevel < requiredVerbosity &&
+      ["/pc ", "/r "].some((cmd) => message.startsWith(cmd))
```

Comment:
I think this is missing a `/w` and/or `/msg` check if we ever switch the implementation of `reply()`